### PR TITLE
Auth Pages: User Interface

### PR DESCRIPTION
**Pull request by vrx29**
This PR just contains the logic for UI for Sign Up and Login pages
- All components are created using Styled components

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+root.render(
   <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById("root")
+    <Router>
+      <GlobalStyle />
+      <App />
+    </Router>
+  </React.StrictMode>
```
Comment (Maianki) [inline src/index.js:19]: appreciation: good that you are using react18

File: src/pages/Login/index.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+const SubHeading = styled.p`
+  font-size: 0.8rem;
+  color: #6c757d;
+`;
+
+const StyledLink = styled(Link)`
+  font-size: 1.1em;
+  font-weight: 600;
+  color: #3554d1;
```
Comment (Maianki) [inline src/pages/Login/index.jsx]: Small suggestion : you can use css variables for colors

**Review by Maianki — APPROVED** [review]
You can go ahead with merge. Very clean and readable code. Good to see that you are using styled componets.

File: src/pages/Login/index.jsx (hunk 2 above)
Comment (vrx29) [reply src/pages/Login/index.jsx]: yup will use them. Once I learn how to use it in styled components

File: src/assets/icons/BackIcon.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export function BackIcon(props) {
```
Comment (Hamza-777) [inline src/assets/icons/BackIcon.jsx:1]: Where do you get these svgs from?
I use react-icons, do you just copy the svg by inspecting them or what?

**Review by Hamza-777 — APPROVED** [review]
Looks good.

File: src/assets/icons/BackIcon.jsx (hunk 3 above)
Comment (vrx29) [reply src/assets/icons/BackIcon.jsx:1]: will ping you the link

**Pull request merged**